### PR TITLE
dependency upgrade

### LINKED ISSUE #40: [Security] Upgrade Undici dependency to latest version
Detailed paths

Introduced through: › apollo-datasource-http@0.21.0 › undici@4.16.0
Fix: Upgrade to undici@5.5.1

Overview

[undici](https://www.npmjs.org/package/undici) is an An HTTP/1.1 client, written from scratch for Node.js

Affected versions of this package are vulnerable to Improper Certificate Validation due to Undici.ProxyAgent missing verification of the remote server's certificate, which leads to exposure of all the requests and responses data to the proxy.

### PR DESCRIPTION
**Pull request by 0xTea**
Hi this pr servers to fix #40  , 35 passed test on npm test . can you please let me know if this will work for undici upgrades

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "apollo-datasource-http",
-  "version": "0.21.0",
+  "version": "0.21.1",
```
Comment (StarpTech) [inline package.json:3]: Please don't bump the version. This is happening automatically.

**Comment by StarpTech** [comment]
Last thing, did you run the benchmarks?

**Comment by StarpTech** [comment]
@0xTea thank you for the PR but I'm going to close this one because https://github.com/StarpTech/apollo-datasource-http/pull/37 is already addressing this.

**Comment by 0xTea** [comment]
> Last thing, did you run the benchmarks?

Yes all benchmarks were run successfully

**Pull request closed without merging**